torquedata/torquedata/routes.py

```diff
@@ -109,20 +114,17 @@ def sheet_toc(sheet_name, toc_name, fmt):
     else:
         raise Exception("Only mwiki format valid for toc")
 
-@app.route('/api/<sheet_name>/id/<key>.<fmt>')
-def row(sheet_name, key, fmt):
-    group = request.args.get("group")
-    wiki_key = request.args.get("wiki_key")
-
-    valid_id = False
 
+def is_valid_id(group, wiki_key, key, sheet_name):
```

Comment:
This method should return False at the end, right?  before `valid_id` was declared to False.  Right now it returns `None` if `wiki_key` isn't in the permissions.